src/js/calculator.js

```diff
@@ -0,0 +1,107 @@
+import { divideOperatorAndNumber } from '../utils/index.js';
+
+class Calculator {
+  constructor({ $target }) {
+    this.$target = $target;
+    this.sum = '';
+    this.operator = '';
+    this.total = document.getElementById('total');
```

Comment:
```suggestion
class Calculator {
  $target;
  sum;
  operator;
  total;
  
  constructor({ $target }) {
    this.$target = $target;
    this.sum = '';
    this.operator = '';
    this.total = document.getElementById('total');
```

클래스 내에서 사용되는 멤버변수들을 상단에 선언해주면 어떨까요? 보다 명시적으로 나타낼 수 있을 것 같아요.

Reply:
제가 잘못이해한것 일 수도 있을 것 같아 질문하나 남겨봅니다 :)
클래스내에서 멤버변수라하시면 instance화 할 때 사용하지 않고 현재 클래스 내에서만 사용할 변수들을 말하시는 게 맞으신가요??
제안해주신 부분에 전체가 동일하게 들어가있어서 혼동이 조금 있었습니다 🥲

Reply:
@sanghunlee-711 

현재 constructor에서 초기화하는 부분이 존재해서 동일하게 들어가 있다고 볼 수 있지만,
앱의 복잡도가 높아지는 경우 constructor에서 초기화하지 않고 메서드에서 초기화하는 경우도 있을거에요. 상속을 사용할 경우 constructor를 작성하지 않는 경우도 있을테구요. 
그런 관점에서 constructor가 아닌 class의 최상위 레벨에서 멤버변수를 선언해주는 것이 좋다고 말씀드린 부분이었습니다!

constructor에서 무조건 멤버변수를 선언한다는 컨벤션을 가져가신다면 이 또한 괜찮을 수 있는 방향이라고 생각합니다